wafw00f/plugins/shieldon.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+'''
+Copyright (C) 2021, WAFW00F Developers.
+See the LICENSE file for copying permission.
+'''
+
+NAME = 'Shieldon Firewall (Shieldon.io)'
+
+
+def is_waf(self):
+    captcha = [
+        self.matchContent('Please solve CAPTCHA'),
+        self.matchContent('shieldon_captcha'),
+        self.matchContent('Unusual behavior detected'),
+        self.matchContent('status-user-info'),
+    ]
+
+    if all(i for i in captcha):
+        return True
+
+    ip_banned = [
+        self.matchContent('Access denied'),
+        self.matchContent('The IP address you are using has been blocked.'),
+        self.matchContent('status-user-info'),
+    ]
+
+    if all(i for i in ip_banned):
```

Comment:
These checks might bring in some false positives, but we'll deal with them, one we get any issues regarding false positives.